content/patterns/layered-zero-trust/lzt-secure-supply-chain.adoc

```diff
@@ -0,0 +1,557 @@
+---
+title: Secure supply chain
+weight: 30
+aliases: /layered-zero-trust/lzt-secure-supply-chain/
+---
+
+:toc:
+:imagesdir: /images
+:_mod-docs-content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="lzt-secure-supply-chain"]
+= Use case: Secure supply chain
+
+[role="_abstract"]
+This use case outlines the process of building, signing, and verifying artifacts and images within the Zero Trust Validated Pattern (ZTVP) using the Red{nbsp}Hat Trusted Artifact Signer (RHTAS) and the Red{nbsp}Hat Trusted Profile Analyzer (RHTPA).
+
+[id="clone-app-source"]
+== Cloning the app source code
+
+The `qtodo` demo application is publicly accessible within its dedicated GitHub repository.
+Clone the repository that has the source code.
+
+.Procedure
+
+* Clone the repository:
++
+[source,terminal]
+----
+$ git clone https://github.com/validatedpatterns-demos/qtodo.git
+----
+
+[id="build-app-artifact"]
+== Building the app artifact
+
+The application requires *Java 17+* and *maven 3.8.x+* to compile.
+However, you can use containers and the Makefile that is already present in the repository.
+
+.Procedure
+
+* To build the application in `uber-jar` format, run the following command:
++
+[source,terminal]
+----
+$ make build
+----
++
+The artifact is built in the `target/` directory.
+
+.Verification
+
+* List the contents of the `target/` directory to verify that the generated JAR file:
++
+[source,terminal]
+----
+$ ls -lh target/
+total 46M
+-rw-r--r-- 1 user user 46M Nov 19 13:42 qtodo-1.0.0-SNAPSHOT-runner.jar
```

Comment:
Should probably update to remove `SNAPSHOT` given the PR on the qtodo repository will remove the use of SNAPSHOT